target/scripts/helpers/log.sh

```diff
@@ -78,12 +78,12 @@ function _log
   case "${1}" in
     ( 'trace' )
       [[ ${LEVEL_AS_INT} -ge 5 ]] || return 0
-      MESSAGE+="  ${LGRAY}TRACE  "
+      MESSAGE+="  ${CYAN}TRACE  "
```

Comment:
Side note: had to adjust this here because, as it turns out, `LGRAY` is never set, and the `set -u` will take care of exiting with error during build in `packages.sh`.